### ISSUE TITLE
LC0017 - Comments does not remove warning

### ISSUE THREAD

**Issue by jwikman** [StefanMaron/BusinessCentral.LinterCop#167]
I get LC0017 on a couple of places where it is intended. The documentation (https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0017) and the description of the warning says that we could remove this warning by adding a comment (just as LC0002). But this does not work for me. 

I've tried with a comment on the line above the warning and in the end of the line of the warning, none of below examples removes the warning:
```
// My Justification
"My FlowField" := WhateverValue;
```

`
"My FlowField" := WhateverValue; // My Justification
`

**Comment by StefanMaron** [comment]
Should be fixed

**Comment by jwikman** [comment]
Yes, this seems to work now! 👍
